### PR TITLE
imapd.c: add support for JMAPACCESS capability and response code

### PR DESCRIPTION
**Pull request by ksmurchison**
Implementation of 
[https://www.ietf.org/archive/id/draft-ietf-extra-jmapaccess-08.html](url)
without a JMAPACCESS capability (as discussed at IETF 119)

### CODE REVIEW

File: changes/next/imap_jmapaccess
Comment (rsto) [inline changes/next/imap_jmapaccess:4]: Unless this gets tagged as `Do Not Merge` until that I-D draft is a standard, should this change log rather call out that this is a prototype implementation?

**Review by rsto — APPROVED** [review]
LGTM, with the caveat that this might rather not get merged in the main branch for now?

File: changes/next/imap_jmapaccess
Comment (ksmurchison) [reply changes/next/imap_jmapaccess:4]: Switched PR to draft until IETF 199

**Comment by brong** [comment]
Yeah, let's deploy this on future and staff and make sure no clients barf!

File: imap/imapd.c
Comment (rsto) [inline imap/imapd.c:420]: typo here

**Review by rsto — APPROVED** [review]
LGTM but please fix the typo before merging

**Pull request merged**
